src/components/Shared/H2.tsx

```diff
@@ -0,0 +1,8 @@
+import styled from "@emotion/styled";
+
+import colors from "../../styles/colors";
+
+export default styled.h2<{ hasBorderBottom?: boolean }>((props) => ({
+  paddingBottom: props.hasBorderBottom ? 8 : 0,
+  borderBottom: props.hasBorderBottom ? `1px solid ${colors.grey}` : 0,
+}));
```

Comment:
I guess same with all of these shared components, these are just pre-styled html tags right?

Reply:
Yea just a reusable component, don't want to keep writing the padding and border style